css/index.css

```diff
@@ -1220,6 +1237,10 @@ img {
     flex: none;
     display: flex;
   }
+
+  .tablet-only {
```

Comment:
What if we need other `display` value here? Also, `-only` suffix usually means that this media query selector will be applied ONLY on specified target, but right now it will be applied both on mobiles and tablets. So, I would either remove `-only` or add `display: none` for this selector on mobiles.
Also, to solve the first issue, we could use inversion:
```
@media (max-width: 767px), (min-width: 992px) {
  .tablet-only {
    display: none;
  }
}
```